bin/run_tertiary_workflow.sh

```diff
@@ -88,19 +69,28 @@ if [ -n "$batch_field" ]; then
 fi
     
 sub_in_params 'cell_type_field' "$cell_type_field"
-sub_in_params 'batch_variable' $batch_field
-sub_in_params 'representation' $representation
-
-run_galaxy_workflow.py -C $GALAXY_CRED_FILE \
-                       -i $inputs_yaml \
-                       -o $WORKDIR \
-                       -W $flavor_dir/scanpy_clustering_workflow.json \
-                       -P $parameters_yaml \
-                       -H scanpy-clustering-$EXP_ID \
-                       -a $flavor_dir/scanpy_clustering_allowed_errors.yaml \
-                       -G $GALAXY_INSTANCE $ADDITIONAL_GALAXY_WF_EXECUTOR_OPTION \
-                       -s $STATE_FILE \
-                       --parameters-yaml
+sub_in_params 'batch_variable' "$batch_field"
+sub_in_params 'representation' "$representation"
+
+
+FLAVOUR_NF=''
+if [ "$FLAVOUR" = 'w_droplet_clustering' ]; then
+    export FLAVOUR_NF='droplet'
+elif [ "$FLAVOUR" = 'w_smart-seq_clustering' ]; then
+    export FLAVOUR_NF='smartseq'
+else
+    echo "Unknown FLAVOUR $FLAVOUR"
+    exit 1
+fi
+
+# Run the workflow
+nextflow run $baseDir/scxa-tertiary-workflow/main.nf \
+    --exp_id $EXP_ID \
```

Comment:
```suggestion
    --dir_path $EXP_ID \
```

Reply:
Here `$EXP_ID` is a relative path where data stored in previous step by `bash $baseDir/scxa-tertiary-workflow/scripts/dataprep.sh $EXP_ID`

Absolute path can be specified too.

Results of the NF wf will be in `$EXP_ID/results`